ext/zend_test/test.stub.php

```diff
@@ -102,3 +102,23 @@ public function method(): void {}
     function namespaced_func(): bool {}
 
 }
+
+namespace ZendTestNS3\ZendSubNS {
+
+    class Foo {
+    }
+
+}
+
+namespace ZendTestNS3 {
+
+    class Foo {
+        public ZendSubNS\Foo $foo;
+
+        public function __construct(ZendSubNS\Foo $foo) {}
+
+        public function method1(ZendSubNS\Foo $foo): void {}
```

Comment:
I don't see why these methods are needed for the test?

Reply:
Yeah, we should just need the property declaration (can also go into one other existing classes above).

Reply:
OK, Fixed it :)